app/utils/admin_action.py

```diff
@@ -131,6 +132,10 @@ def _admin_action_is_similar(action_obj1, action_obj2):
 
 
 def send_slack_notification(new_status, action_obj, action_service: Service):
+    if _is_request_from_functional_tests():
+        current_app.logger.info("Skipping sending SlackMessage because this was from functional tests", action_obj)
+        return
```

Comment:
Maybe call this function (and the variables around it) should_prevent_slack_alerting or something, then in the config where the IP's are denoted mark them as from the functional tests?

Reply:
Hmm, I'm not too sure. You'd presumably save one line of code at best as you'll need to return for control flow? (Or could invert it and only run if the `if` passes, but I prefer happy control path and early return IMO)

E.g.
```python
def send_slack_notification(new_status, action_obj, action_service: Service):
    if should_prevent_slack_alerting():
        return
```

Reply:
It's mostly the name rather than saving code, so it can be used for other purposes rather than functional tests if required